PC/layout/support/options.py

```diff
@@ -55,6 +55,10 @@ def public(f):
         "help": "nuget package",
         "options": ["stable", "pip", "distutils", "dev", "props"],
     },
+    "iot": {
+        "help": "Windows IoT Core",
+        "options": ["stable", "pip", "distutils", "dev", "props"],
```

Comment:
If cross-compilation is necessary, is there any value in including distutils and dev? Those are only useful when building packages. (When cross-compiling, I'd expect users to pick up the nuget package rather than the IoT package)

Reply:
Good idea.  When I was testing this out yesterday I was thinking it would be nice to have different packages for cross-compiling and on-target deployment, but I didn't think of using the nuget package.